packages/flutter/test/material/dropdown_menu_test.dart

```diff
@@ -4279,6 +4279,28 @@ void main() {
       expect(iconButton, findsNothing);
     },
   );
+
+  testWidgets('The controller provided to DropdownMenu should be able to call dispose.', (
```

Comment:
I suggest the test name written more explicitly, such as
```suggestion
  testWidgets('Explicitly provided controllers should not be disposed when switched out.', (
```